services/email.js

```diff
@@ -9,9 +9,11 @@ export default {
     name: 'email',
     // At least one of the CRUD methods is Required
     create: (req, resource, params, body, config, callback) => {
-        if (!params.from || !params.to || !params.subject || !params.message)
+        if (!params.subject || !params.message)
             return callback('Error: Missing parameters', null);
 
+        const to = SMTP.to, from = SMTP.from;
```

Comment:
My thinking:
_From_ and _To_ have to be given by the config: _To_ relates to the email address where we want that each _contact us_ email should go. _From_ is the email address which is used as sender which just could be something@slidewiki.org, because imitating other email addresses will result in marking our domain globally as SPAM.
The action/component have to make sure that the message of the email contains everything. This is already done by Sole in the Contact Us component.
Using SMTP _To_ and _From_ is well defined but yes I could use longer names.